tests/scripts/test_scripts.py

```diff
@@ -650,7 +650,7 @@ def test_preference_comparisons_transfer_learning(
         tmpdir: Temporary directory to save results to.
         named_configs_dict: Named configs for preference_comparisons and rl.
     """
-    tmpdir_path = pathlib.Path(tmpdir)
+    tmpdir_path = types.parse_path(tmpdir)
```

Comment:
What was wrong with just using `pathlib.Path` here given `tmpdir` is guaranteed to be a `str`?